src/Microsoft.Health.Fhir.Core/Features/Validation/ResourceElementValidator.cs

```diff
@@ -3,23 +3,23 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 using FluentValidation;
+using FluentValidation.Validators;
 using Microsoft.Health.Fhir.Core.Features.Validation.FhirPrimitiveTypes;
 using Microsoft.Health.Fhir.Core.Features.Validation.Narratives;
 using Microsoft.Health.Fhir.Core.Models;
 
 namespace Microsoft.Health.Fhir.Core.Features.Validation
 {
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = "Follows validator naming convention.")]
-    public class ResourceElementValidator<T, TProperty> : AbstractValidator<ResourceElement>
+    public class ResourceElementValidator<T> : AbstractValidator<ResourceElement>
```

Comment:
Doesn't look like T is used anymore

Reply:
I agree, looks like al *validator classes can be non generic, and just pass ResourceElement to thier base generic class.

Reply:
Updated Validators :)